subprojects/docs/src/samples/java/incubating/jvm-multi-project-with-additional-test-types/kotlin/application/build.gradle.kts

```diff
@@ -0,0 +1,13 @@
+plugins {
+    id("myproject.java-conventions")
+    application
+}
+
+dependencies {
+    implementation(project(":list"))
+    implementation(project(":utilities"))
```

Comment:
I think this is the same sample project as with the other multi-project applications. Do you know why this sometimes has an implementation dependency on utilities AND list and sometimes it's just utilities? e.g., see the modules-multi-project-with-integration-tests sample below.

Reply:
I think this is because in this project the `"utilities` project had only an `implementation` dependency on `:list` until we updated it on 8/19.  I'll get these to only dep `:utilities`.